charts/hedgedoc/README.md

```diff
@@ -33,29 +41,34 @@ By default, this Helm Chart deployed a postgreSQL database along Hedgedoc. If yo
 | `hedgedoc.podSecurityContext`          | Set SecurityContext for the pod                         | `{}`                                       |
 | `hedgedoc.containerSecurityContext`    | Set SecurityContext for the container                   | `allowPrivilegeEscalation: false`<br>`runAsUser:`<br>`runAsGroup:`<br>`capabilities:`<br>&nbsp;&nbsp;`drop:`<br>&nbsp;&nbsp;`- ALL`<br>`runAsNonRoot: true`<br>`seccompProfile:`<br>&nbsp;&nbsp;`type: RuntimeDefault` |
 | `hedgedoc.pvc.storageSpace`            | Storage space for the PersistentVolume                  | `5Gi`                                      |
-| `hedgedoc.service.type`                | Set the Service type                                    | `ClusterIP`                                |
 | `hedgedoc.random_pw_secret_key`        | Key to store the password                               | `database-password`                        |
-| `hedgedoc.secret.database-name`        | Name of the database                                    | `postgres`                                 |
-| `hedgedoc.secret.database-user`        | Name of the postgres user                               | `postgres`                                 |
-| `hedgedoc.secret.database-password`    | Function that retrieve the generated password           | `'{{- include "random_pw_reusable" . -}}'` |
+| `hedgedoc.secret.databaseName`         | Name of the database. Linked to `postgresql.auth.database`. EDIT `postgresql.auth.database` instead of this value!                                    | `{{ tpl .Values.postgresql.auth.database . }}` |
+| `hedgedoc.secret.databaseUser`         | Name of the postgres user. Linked to `postgresql.auth.username`. EDIT `postgresql.auth.username` instead of this value!                              | `{{ tpl .Values.postgresql.auth.database . }}` |
+| `hedgedoc.secret.databasePassword`     | Empty by default because the password is generated randomly and stored in a Secret. Replace this value if you want to add your custom password. | `''` |
+| `hedgedoc.secret.password`     | Empty by default because the password is generated randomly and stored in a Secret. Replace this value if you want to add your custom password. | `''` |
+| `hedgedoc.secret.postgresPassword`     | Empty by default because the password is generated randomly and stored in a Secret. Replace this value if you want to add your custom password. | `''` |
 
 ### PostgreSQL parameters
 
 Since we are using the `bitnami/postgresql` Helm Chart as a dependency, you can take a look to the [PostgreSQL ArtifactHub](https://artifacthub.io/packages/helm/bitnami/postgresql/15.5.0) to check the different values.
 
-The postgres database password is generated randomly and won't change if you upgrade the Chart.
+This parameters is disabled by default. If you want to use a postgresql database among, enable it with `postgresql.enabled`
```

Comment:
Small grammar mistake. It is either `These parameters` or `This parameter`

Reply:
Fixed